microsoft/testsuites/display/drm_sanity.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from typing import Any, cast
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+
+from lisa.base_tools import Cat
+from lisa.tools import Lsmod
+
+@TestSuiteMetadata(
+    area="drm",
+    category="functional",
+    description="""
+    This test suite uses to verify drm driver sanity.
+    """,
+    requirement=simple_requirement(unsupported_os=[]),
+)
+class DRM_Sanity(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This case is to check whether the drm driver registered successfully.
+
+        Step,
+        1. Check hyperv_drm driver is inserted correctly.
```

Comment:
Give more details, how it's checked. The test metadata will be abstracted out of the code. It needs to help understand details.

Reply:
ok